static/css/styles.css

```diff
@@ -43,6 +43,7 @@ nav li {
 }
 nav li a,
 nav li .dropdown-target {
+	cursor: pointer;
```

Comment:
What's the reasoning behind adding this style? IMO it makes the target behave more like a clickable link.

Reply:
Good point! My thinking was that the target still behaves like a menu button and should get the pointer cursor rather than the generic text selection cursor to depict it as clickable.

Reply:
The target doesn't do anything on click, so a pointer cursor sends a mixed message. Besides, by the time the user hovers over it to change the cursor, the popup is already open.